abis/Federation.json

```diff
@@ -261,6 +274,19 @@
         "stateMutability": "view",
         "type": "function"
     },
+    {
+        "inputs": [],
+        "name": "bridgeNFT",
```

Comment:
maybe NFTBridge? dunno

Reply:
this would affect other names below probably (I just think it reads better, but we could keep it this way)

Reply:
Yeah, could be NFTBridge 
Actually, i put NFTBridge first, but it's doesn't seem like camelCase haha so I changed to bridgeNFT

Reply:
maybe we could do this? kinda weird but seems legit
https://stackoverflow.com/a/15526526/3922574

Reply:
would end up being NftBridge everywhere I guess (maybe I'd leave that for an individual PR)

Reply:
I like that 👍